react.js

```diff
@@ -61,6 +61,8 @@ module.exports = {
     'react/jsx-uses-vars': 'warn',
     // Prevent missing parentheses around multilines JSX
     'react/jsx-wrap-multilines': 'warn',
+    //  Prevent using Array index in key props
+    'react/no-array-index-key': 'warn',
```

Comment:
I can see it being overridden at a project level fairly often, which may help inform what it should be here over time. In that case I'm fine leaving it on by default, and we can re-evaluate if necessary.